_includes/connect-zbt-1/migrate-zha-only.md

```diff
@@ -1,71 +1,132 @@
 <!---steps on migrating an existing ZHA network to a Home Assistant Connect ZBT-1 stick. -->
 
+{% steps %}
+{% step "Starting radio migration" %}
+{% image "/static/img/connect-zbt-1/connect-zbt-1-z2m-migrate-zha-02.png" "Screenshot showing the migrate radio button on the Network settings dialog" %}
+{% stepContent %}
 
-1. Starting radio migration.
-   1. Under [**Settings** > **Devices & services**](https://my.home-assistant.io/redirect/integrations/), select the ZHA integration, then select **Configure**.
-   2. Under **Network settings** add-on, select **Migrate Radio**.
+1. Under [**Settings** > **Devices & services**](https://my.home-assistant.io/redirect/integrations/), select the ZHA integration, then select **Configure**.
+2. Under **Network settings** add-on, select **Migrate Radio**.
 
-   ![Screenshot showing the migrate radio button on the Network settings dialog](/static/img/connect-zbt-1/connect-zbt-1-z2m-migrate-zha-02.png)
+{% endstepContent %}
+{% endstep %}
 
-2. To reconfigure ZHA, select **Submit**.
+{% step "Reconfiguring ZHA" %}
+{% image "/static/img/connect-zbt-1/z2m-migrate-zha-03.png" "Screenshot showing the Reconfigure ZHA dialog" %}
+{% stepContent %}
 
-    ![Screenshot showing the Reconfigure ZHA dialog](/static/img/connect-zbt-1/z2m-migrate-zha-03.png)
+- To reconfigure ZHA, select **Submit**
```

Comment:
```suggestion
- To reconfigure ZHA, select **Submit**.
```

Reply:
thank you for the catch! those are covered in another PR. don't remember which, but they are fixed now.